extensions/stackdriver/log/logger.cc

```diff
@@ -44,15 +44,18 @@ Logger::Logger(const ::wasm::common::NodeInfo& local_node_info,
       std::make_unique<google::logging::v2::WriteLogEntriesRequest>();
 
   // Set log names.
-  const auto& project_id =
-      local_node_info.platform_metadata().at(Common::kGCPProjectKey);
+  const auto& platform_metadata = local_node_info.platform_metadata();
+  auto iter = platform_metadata.find(Common::kGCPProjectKey);
+  std::string project_id = "";
+  if (iter != platform_metadata.end()) {
+    project_id = iter->second;
+  }
   log_entries_request_->set_log_name("projects/" + project_id + "/logs/" +
```

Comment:
maybe we should skip setting log name if project ID is not found? or is there a better default?

Reply:
There is no good fallback, we should just use the validation API to reject the listener config. I added a TODO for it, will do it in another PR.